docker-entrypoint/dependencies/entrypoint.go

```diff
@@ -0,0 +1,80 @@
+package entrypoint
+
+import (
+	"github.com/stackanetes/docker-entrypoint/logger"
+	//	"k8s.io/kubernetes/pkg/client/restclient"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+	"os"
+	"sync"
+	"time"
+)
+
+var (
+	Dependencies     []Resolver      // List containing all dependencies to be resolved
```

Comment:
Don't expose it, it should only be manipulated by Register. Also, prefer using a map with the name, so we can avoid registering twice by mistake.

Example:
```
func Register(name string, resolver Resolver) {
	if driver == nil {
		panic("dependencies: could not register nil Resolver")
	}
	if _, dup := resolvers[name]; dup {
		panic("dependencies: could not register duplicate Resolver: " + name)
	}
	resolvers[name] = resolver
}
```